packages/cli/lib/lib/webpack/webpack-client-config.js

```diff
@@ -156,14 +158,12 @@ async function clientConfig(env) {
 	};
 }
 
-function getBabelEsmPlugin(config) {
```

Comment:
Besides a few spots in `client-config`, everywhere else we use `env`, rather than `config`.